internal/pkg/deploy/cloudformation/stack/worker_svc.go

```diff
@@ -139,7 +140,11 @@ func (s *WorkerService) Template() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("parse worker service template: %w", err)
 	}
-	return content.String(), nil
+	overridenTemp, err := s.taskDefOverrideFunc(convertTaskDefOverrideRules(s.manifest.TaskDefOverrides), content.Bytes())
```

Comment:
nit: Temp is usually used for "temporary" 😄 
```suggestion
	overridenTpl, err := s.taskDefOverrideFunc(convertTaskDefOverrideRules(s.manifest.TaskDefOverrides), content.Bytes())
```